h2o-docs/src/product/data-science/glrm.rst

```diff
@@ -149,6 +180,9 @@ Below is a simple example showing how to build a Generalized Low Rank model.
                                                 transform="standardize")
     glrm_model.train(training_frame=train) 
 
+    # Transform the data using the validation set to retrieve the new coefficients:
+    glrm_transform = glrm_model.transform_frame(valid)
+
```

Comment:
@hannah-tillman Can you change the wording:

using the validation set

to

using the dataset valid